api/application.ts

```diff
@@ -38,6 +39,12 @@ if (showFeature(FEATURE_HELMET_ENABLED)) {
   app.use(helmet.xssFilter());
   app.use(getContentSecurityPolicy(helmet));
   app.use((req, res, next) => {
+    const origin = app.get('env') === 'development' ? 'http://localhost:3001' : 'https://manage-case.aat.platform.hmcts.net';
```

Comment:
That needs to be smarter - obviously that's only going to work on local and aat - needs to handle production, preview and demo, ithc and perftest environments